### ISSUE TITLE
Invalid process definition -- Unknown keyword 'stub' @ line 20, column 5

### ISSUE THREAD

**Issue by bioramg** [ctmrbio/BACTpipe#164]
Hi,
I installed dependencies as per suggestion. After the instalment, I run to check...
```
$nextflow run tutorial.nf
N E X T F L O W  ~  version 20.10.0
Launching `tutorial.nf` [modest_snyder] - revision: e3b475a61b
executor >  local (3)
[ee/6f3599] process > splitLetters       [100%] 1 of 1 ✔
[84/ac6b19] process > convertToUpper (1) [100%] 2 of 2 ✔
WORLD!
HELLO
```

Then, I tried to assemble my bacterial genomes. But I am getting an error like this:
```
$nextflow run ctmrbio/BACTpipe --reads /home/2021/tri_mt/trimming/tri_mt.fq.gz
N E X T F L O W  ~  version 20.10.0
Launching `ctmrbio/BACTpipe` [confident_mclean] - revision: 36d0bad23d [master]
============================================================
BACTpipe
Version 3.0
Bacterial whole genome analysis pipeline
https://bactpipe.readthedocs.io
============================================================
Module compilation error
- file : /home/raman/.nextflow/assets/ctmrbio/BACTpipe/./modules/assembly_stats/assembly_stats.nf
- cause: Invalid process definition -- Unknown keyword `stub` @ line 20, column 5.
       """
       ^

1 error
```

What could be the problem and how to solve it?

Thank you.

**Comment by abhi18av** [comment]
Hi @bioramg ,

Thanks for reporting this issue, the `stub` is a new feature in Nextflow and you can read more about this [here](https://www.nextflow.io/docs/edge/process.html?highlight=stub#stub).

The problem you're facing could be solved by installing the latest NF version you could refer this comment here https://github.com/ctmrbio/BACTpipe#quick-start .

Warmly,
Abhinav

**Comment by boulund** [comment]
@abhi18av do we have any information on when a new release of Nextflow is coming that will include the `stub` feature? This has bitten be several times as well, really frustrating 😄

**Comment by abhi18av** [comment]
Yeah, I can understand. 

The good news is that the new stable release is out already `21.04` https://github.com/nextflow-io/nextflow/releases/tag/v21.04.0

NOTE: NF generally makes a stable release in April and October (similar to Ubuntu) in addition to the monthly `edge` releases.

**Comment by boulund** [comment]
Fantastic 🎉 
Then I guess we should update the README accordingly, could you push such a commit to the dev branch so it's merged with the rest of the changes in #160 ?

**Comment by abhi18av** [comment]
Sure, I've added it here https://github.com/ctmrbio/BACTpipe/pull/165/commits/2300472a6d80493a95393776079bf64aca611923 

Also, closing the issue since the instructions are updated.
